doc/rst/developer/bestPractices/CompilerDebugging.rst

```diff
@@ -103,7 +112,7 @@ print_view(BaseAST*)  # view, vi    # not showing ID
 nprint_view(BaseAST*) # nview, nv
```

Comment:
Similar to below, I'd just toss this whole section (from line 109 to 120) into a code-block to preserve the intended spacing and avoid having to escape any syntax.

Reply:
do I not include ``legend: C function # gdb shortcut`` - line 108, in the block? It seems thought it might be something that should be in tbe block.